database/redis.go

```diff
@@ -29,7 +29,7 @@ func GetRedisUrl() (redis.Conn, error) {
 		logs.Critical("Redis url not found: ", err)
 	}
 
-	return redis.Dial("tcp", redisUrl)
+	return redis.DialURL(redisUrl)
```

Comment:
`Dial` does not work with the Heroku REDIS_URL

Reply:
@gutakk The error was `too many colons in address` and it cant connect with the `redis`

Reply:
In case you still want to use redis, maybe you can take a look at my [redis connection ](https://github.com/gutakk/go-google-scraper/blob/master/db/redis.go#L29). Redigo also cannot connect to heroku redis with the same error as yours.

Reply:
Yeah, I've heard some issue about `redis connection` issue from Hoang also.
Maybe Hoang might know the solution 🍀

Reply:
Here is our post discussion cc @Lahphim 
![Screen Shot 2021-04-09 at 3 15 13 PM](https://user-images.githubusercontent.com/6076762/114150579-68cda480-9946-11eb-8c0a-40ce0e92939b.png)

Reply:
Yes, I faced this and solved it by adding [redisurl](github.com/soveran/redisurl) to get the correct information (Gut suggested me to use it).
But good to know that `redigo` has `DialURL()` and it did the same thing like `redisurl` 😄

Reply:
@hoangmirs Yes Hoàng , I found it in [here](https://github.com/gomodule/redigo/issues/257#issuecomment-308867661) to using `DialURL()` 😄